_targets.R

```diff
@@ -0,0 +1,35 @@
+library(targets)
+source("code.R")
```

Comment:
This is an interesting approach! I usually just source all the additional R scripts right here. I've never considered organizing it in the way you are doing here 👍